core/src/main/java/nl/nn/adapterframework/senders/JavascriptSender.java

```diff
@@ -68,17 +66,17 @@ public class JavascriptSender extends SenderSeries {
 	private String fileInput;
 
 	public enum JavaScriptEngines {
-		J2V8(J2V8.class), NASHORN(Nashorn.class), RHINO(Rhino.class);
+		J2V8(J2V8.class);
```

Comment:
Does it make sense to maintain support for multiple JS engines?
Or is the effort to take that out too great?

Should we do anything to facilitate loading old configurations with values of "Rhino" or "Nashorn" in here, which are now no longer valid enum values?

I think that unless we foresee adding new JS engines in future we should at least have a separate PR for 7.9 to make the `engine` config property of the JavascriptSender `@Deprecated` and with a `@ConfigurationWarning` attached.

Reply:
Done inside #5517